### ISSUE TITLE
Update Content Migrator service to use latest Unity Renderer

### ISSUE THREAD

**Issue by pravusjif** [decentraland/unity-renderer#200]
- [x] Update [ABs Playbook](https://playbooks.decentraland.systems/world-team/asset-bundles.html) to mention how to upload wearables manually to the S3 bucket and to mention our ABs Optimizations (loading speed + no-skeleton on non-bodyshapes + giant tex resize + mesh optimization)
- [ ] ~Upload an updated docker image to dockerhub using the latest AB Converter changes from the repo master branch~
- [ ] ~Update consumer & producer services/tasks at AWS ECS to use the new version~
- [ ] ~Update graphana metrics to read from the new version~
- [ ] ~Include yaml to build docker image in CI~
- [x] Update content-migrator repo to point to our new Unity2020 docker image
- [x] Update explorer submodule to point to the latest updates (change it to point to the new unity-renderer repo)
- [ ] ~Activate Unity license in new docker image (build docker image locally, create a .env file with the repo's environment variables, run the image in local, use the image terminal to [generate the unity license activation request xml and activate it manually](https://forum.unity.com/threads/command-line-manuallicensefile-on-linux.889129/#post-6239651), to register the docker image VM hardware in the license)~
- [x] Update [ABs Playbook](https://playbooks.decentraland.systems/world-team/asset-bundles.html) changing the manual docker image building instructions, since now the content-migrator repo CI should do all the work.

~Follow [playbook guidelines](https://playbooks.decentraland.systems/world-team/asset-bundles.html) to clone migrator repo, create updated docker image, upload the image to docker hub and update the services to use the new image through pulumi (and the grafana metrics have to be manually updated as well)~

**Comment by pravusjif** [comment]
Our Unity2020 docker image is breaking the content-migrator docker image building.

We should compare the new Unity2020 image with the previous "gableroux" image used and see what's missing in the new one.

**Comment by pravusjif** [comment]
Local docker image building fixed with support from @kuruk-mm & @BrianAmadori.

Pushed changes to the gitlab content-migrator repo, apparently that alone should trigger the docker imager building, deploying and the pertinent updates in the AWS services. Updated issue checklist accordingly.

If everything works fine in the CI pipeline I'll also update the ABs Playbook to correct the parts that say we have to do it manually.

**Comment by pravusjif** [comment]
With the help of @menduz we've been able to update the content-migrator repo and CI to point to the unity-renderer latest changes.

We also improved the Visual Test step of the AB conversion exit code reporting.

Right now I'm dealing with the Unity license not working in the new docker image, so I'll try the following steps recommended by @BrianAmadori :
1. build docker image locally
2. Create a .env file with the repo's environment variables
3. Run the image in local usign the .env file (the repo readme has the command for that)
4. [Use the image terminal to generate the unity license activation request xml](https://forum.unity.com/threads/command-line-manuallicensefile-on-linux.889129/#post-6239651)
5. Activate the license manually, to register the docker image VM hardware in the license

**Comment by pravusjif** [comment]
Apparently the license activation was not needed since during the weekend the new logs of the Consumer service showed that the current licence was accepted, even though last friday logs had license errors, although the service was probably not started correctly last week since it stopped working during the weekend.

We restarted both services (producer and consumer) and now the converter has been working again since then.

We only see some, apparently harmless, errors in the [Producer logs](https://grafana.decentraland.systems/d/UzieKAoGk/unity-asset-bundle-creation?orgId=1&refresh=5m&from=now-1h&to=now) now.

I entered one of the scene conversions ending with an exit code 4 (some assets conversion were skipped), tried the conversion locally with Unity editor and had the exact same results. Then I added these debugging instructions to the ABs Playbook.

I also checked that new assets were deployed to the S3 bucket since yesterday's service restart.
